DTAdmin/Subject/LogViewController.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  LogViewController.swift
+//  DTAdmin
+//
+//  Created by ITA student on 10/20/17.
+//  Copyright © 2017 if-ios-077. All rights reserved.
+//
+
+import UIKit
+
+class LogViewController: UIViewController {
+
+    
+    @IBOutlet weak var loginTextField: UITextField!
+    
+    @IBOutlet weak var passwordTextField: UITextField!
+    
+    let queryService = QueryService()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+    
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+    }
+    
+    
+    private func showMessage(message: String) {
+        let alert = UIAlertController(title: NSLocalizedString("Warning", comment: "Alert title"), message: message, preferredStyle: .alert)
+        alert.addAction(UIAlertAction(title: NSLocalizedString("Ok", comment: "Ok button"), style: .default, handler: nil))
+        self.present(alert, animated: true, completion: nil)
+    }
+    
+    @IBAction func logIn(_ sender: UIButton) {
+        let username = loginTextField.text
+        let password = passwordTextField.text
+        
+        if let user = username, let psw = password {
```

Comment:
It's better to check for empty string. (e.g. `!username?.isEmpty`). Do the same for a password.